node/src/main/scala/com/wavesplatform/settings/package.scala

```diff
@@ -49,6 +50,14 @@ package object settings {
 
   implicit val privateKeyReader: ValueReader[PrivateKey] = byteStrReader.map(PrivateKey(_))
 
+  implicit val addressReader: ValueReader[Address] =
```

Comment:
Not sure this works for custom networks. In `Application` we load a config, parse settings and only then set a network byte:
```scala
// Application.scala
    val settings = WavesSettings.fromRootConfig(config) // (1)

    // Initialize global var with actual address scheme
    AddressScheme.current = new AddressScheme {
      override val chainId: Byte = settings.blockchainSettings.addressSchemeCharacter.toByte
    }
```
So we should get a network byte and setup the address scheme before we parse settings (1).

Reply:
Fixed